src/Wallet.purs

```diff
@@ -30,15 +26,24 @@ import Data.Tuple.Nested ((/\))
 import Effect (Effect)
 import Effect.Aff (Aff)
 import Helpers ((<<>>))
-import Serialization.Types (PrivateKey)
+import Wallet.Cip30 (Cip30Wallet, Cip30Connection) as Cip30Wallet
+import Wallet.Cip30 (Cip30Wallet, Cip30Connection, mkCip30WalletAff)
+import Wallet.Key
+  ( KeyWallet
+  , PrivatePaymentKey
+  , PrivateStakeKey
+  , privateKeysToKeyWallet
+  )
+import Wallet.Key (KeyWallet, privateKeysToKeyWallet) as KeyWallet
 
 data Wallet
   = Nami Cip30Wallet
   | Gero Cip30Wallet
   | KeyWallet KeyWallet
 
-mkKeyWallet :: PrivateKey -> Wallet
-mkKeyWallet = KeyWallet <<< privateKeyToKeyWallet
+mkKeyWallet :: PrivatePaymentKey -> Maybe PrivateStakeKey -> Wallet
+mkKeyWallet payKey mbStakeKey = KeyWallet $ privateKeysToKeyWallet payKey
+  mbStakeKey
```

Comment:
optional:
```
mkKeyWallet :: PrivatePaymentKey -> Maybe PrivateStakeKey -> Wallet
mkKeyWallet payKey = KeyWallet <<< privateKeysToKeyWallet payKey
```

Reply:
I try to avoid point-free style when there is more than one argument, because it looks like a mixture of point-free and normal style.